docs/pipelineruns.md

```diff
@@ -1392,17 +1392,18 @@ Your `PipelineRun`'s `status` field can contain the following fields:
 
 ### Configuring usage of `TaskRun` and `Run` embedded statuses
 
-Currently, the default behavior is for the statuses of `TaskRun`s and `Run`s within this `PipelineRun`
-to be embedded in the `status.taskRuns` and `status.runs` fields. This will change in the future to
-instead default to `status.childReferences` being populated with references to the `TaskRun`s and
+Currently, the default behavior is default to `status.childReferences` being populated with references to the `TaskRun`s and
 `Run`s, which can be used to look up their statuses.
 
 This behavior can be controlled by changing the `embedded-status` feature flag in the `feature-flags`
 config map. See [`install.md`](./install.md#customizing-the-pipelines-controller-behavior) for more
 information on feature flags. The possible values for `embedded-status` are:
-- `full` - The current default behavior of populating `status.taskRuns` and `status.runs`, without populating `status.childReferences`.
-- `minimal` - Just populate `status.childReferences`, not `status.taskRuns` or `status.runs`.
+- `minimal` - The current default behavior, populate `status.childReferences`, not `status.taskRuns` or `status.runs`.
 - `both` - Populate `status.childReferences` as well as `status.taskRuns` and `status.runs`.
+- `full` - Populating `status.taskRuns` and `status.runs`, without populating `status.childReferences`.
+
+*Note that after the PipelineRunStatus migration as planned in [TEP-100](https://github.com/tektoncd/community/blob/main/teps/0100-embedded-taskruns-and-runs-status-in-pipelineruns.md),
```

Comment:
NIT: PipelineRunStatus -> `PipelineRunStatus` 

The link points to the whole TEP, perhaps it could point to a specific section? It's not clear to me what is meant by "PipelineRunStatus migration".  I think the other values of the flag will be deprecated and eventually removed. 

Now that the default is set to "minimal", shouldn't we deprecate the other values? They should be added to https://github.com/tektoncd/pipeline/blob/main/docs/deprecations.md and we could link to that document instead

Reply:
Yes, indeed `status.taskruns` and `status.runs` are in the deprecation list, which should be or else we could not remove them in the next release :)

About feature flags, in the TEP we said that we cannot remove flags associated to changes in behaviour - see https://github.com/tektoncd/community/blob/main/teps/0033-tekton-feature-gates.md#changes-in-behavior - but this change of behaviour is reflected into the API, so it doesn't make sense to keep the flag around once the API field it controls is deprecated and removed.

That said, we still should give notice to our users about upcoming changes, so we should include in the release notes and ideally in https://github.com/tektoncd/pipeline/blob/main/docs/deprecations.md too that this flag is deprecated, before we remove it in the next release.

Reply:
Thanks @afrittoli ! That makes sense, updated and linked in deprecation.md.